server/store/api/src/main/java/com/alipay/sofa/registry/store/api/elector/AbstractLeaderElector.java

```diff
@@ -39,9 +39,9 @@ public abstract class AbstractLeaderElector implements LeaderElector {
 
   private volatile LeaderInfo leaderInfo = LeaderInfo.HAS_NO_LEADER;
 
-  private volatile boolean startElector = false;
```

Comment:
The get method of startElector is not synchronized, so I think the volatile keyword for this variable should not be removed.

Reply:
yes, my bad. startElector should not remove volatile.

fixed up.